specifyweb/frontend/js_src/lib/utils/__tests__/hexToHsl.test.ts

```diff
@@ -0,0 +1,39 @@
+import { hexToHsl } from '../utils';
+
+describe('hexToHsl', () => {
```

Comment:
Looking very good!

This style of tests (with `describe` and `test`) is the most common and most flexible. It allows for testing asynchronous functions, mocking functions, and having multiple `expect()` assertions.

That is why it's good to know how to write this style of tests.
However, to make writing tests simpler and quicker, we have an alternative style - using a `theories` function - that will allow you to replace the entire `describe` blog with just ~7 lines!

Please take a look at our existing tests to see how `theories` is used. I also pushed a commit on this branch that improved `theories` documentation.

Main advantage of `theories` style is how much smaller the test definitions are (and thus it takes less time to write them). Also, you don't have to write a test case names, as `theories` generates test case names on the fly for you! But, they are limited in features. Including:
- Don't integrate with IDE test running plugins as well
- Don't support async functions
- Don't support multiple assertions
- Don't support mocking (unless doing a mock for an entire file, rather than a single test)

Thus, it's best to know both styles and use whichever is most appropriate in a given case